### ISSUE TITLE
Default choice of law?

### ISSUE THREAD

**Issue by proftomwbell** [kleros/policies#7]
Most comprehensive contracts specify both what law will govern disputes arising under the contract (choice of law) and the mechanism for resolving such disputes (choice of forum).  Kleros presumably has users agree, at some point, to have their disputes decided in the system.  But does it specify the choice of law?

Sometimes parties with disputes in Kleros will already have chosen the applicable law, perhaps by separate agreement.  But some parties using Kleros will not have thought much about what laws will govern their disputes with counterparties.  And if their transaction occurred across jurisdictional borders, the default choice of law will not be at all obvious.

Kleros should thus specify a default choice of law for conflicts resolved in its system.  This will make the service more reliable for parties who, coming from different countries, do not necessarily agree on what law should govern their dispute.  But what default law should Kleros use?

The Kleros default should avoid seeming to favor any particular place, lest it fail to offer litigants true objectivity.  It should also look for a body of law that users can trust, because it comes from time-tested precedents and long consideration by judges, lawyers, and scholars.  And so that the default choice of law should change with user demands, it should be open source.

Check out [Ulex, the open source legal system](https://github.com/proftomwbell/Ulex).  It offers a comprehensive set of rules from the most respected flag-free sources.  If parties before Kleros had an contract claim, for instance, the American Law Institute's "Restatement" of the common law of contracts would by default govern their dispute.

In many cases, granted, the jurors might resort to principles more universal and atavistic to settle disputes.  In some cases, though, parties before Kleros might want more detailed rules applied.  For them, Kleros should offer a default choice of law.  Kleros could use or adapt Ulex for that purpose.

**Comment by epiqueras** [comment]
We could have a policy that basically says "follow all the rules of Ulex", and then let a subcourt's governance system include it in the subcourt's list of policies.

What do you think, @clesaege ?

At first thought, I think the type of disputes that will require something like Ulex, will also be the type of disputes that Kleros is the least effective with.

**Comment by clesaege** [comment]
Yeah, it should be good to have kind of default corpus to backtrack on even if the first usecases won't probably need it.
You can propose a policy for that @proftomwbell

**Comment by proftomwbell** [comment]
Glad to help--but can you clarify how?  I thought I'd initiate a pull request on your default user agreement, but cannot find that anywhere.  Or maybe you just want some standard issue boilerplate; if so, this should work:  ""Ulex 1.1, available at https://goo.gl/YByFti, governs any claim or question arising under or related to this agreement, including the proper forum for resolving disputes, all rules applied therein, and the form and effect of any judgment."

**Comment by clesaege** [comment]
The Kleros cooperative does set the rules, we can publish draft and propose policies, but ultimately they will be voted through the governance mechanism, so you are free to create a policy proposal.
About Ulex, we may want to specify what applies and what not, some elements are about who make the decisions which is not something which fit into policies (the Kleros ruling mechanism is smart contract enforced, so for example "Judges: Each of the opposing parties chooses a judge, and those two judges choose a third." cannot apply).

**Comment by proftomwbell** [comment]
Good eye, @clesaege ; Ulex's default procedure does not map 1-1 to the default Kleros one.  Happily, Ulex includes meta-rule 3.1 giving priority to later-added rules.  So Kleros could simply append its own procedure to ensure it controls.

Even simpler, it could simply add to any clause invoking Ulex as the default substantive rule (as above) an additional stipulation that local procedural rules will apply.  To wit:  "Ulex 1.1, available at https://goo.gl/YByFti, governs any claim or question arising under or related to this agreement, including the proper forum for resolving disputes, all rules applied therein, and the form and effect of any judgment, except insofar as it conflicts with Kleros' procedural rules, which shall in that event control."

**Comment by bionexus** [comment]
I gave Ulex a look and the thing I don't get is where I can get the rules themselves. You compiled a list of law restatements that help resolving disputes. But how do jurors get these restatements? The tort restatement you proposed costs $ 250 on the ALI website. How shall jurors pay that? Or am I missing some link to a free source?

**Comment by proftomwbell** [comment]
You ask, @bionexus, how you can find the laws of Ulex.  Quick answer:  Same way you get most rules in most legal systems: by looking up what you can and, if you're smart and careful, relying on a lawyer for real answers.  Ulex at least leaves you no worse off on that count than a conventional legal ysstem, and considerably better once you have the full picture in mind.

How, for example, do you know what law to apply to a gig contract by a California-based company with an Arizona-based designer?  Pretty quickly, you would have to get professional help.  First, to figure out what law applies.  Setting Ulex as the default for disputes arising on Aragon would solve that particular problem right out of the gate.

Second, if and when you figure out which state's law applies, you will probably try searching the web for statutes and cases from that jurisdiction.  You might find some relevant legislation and regs online, but you will not likely be very sure you have found the right rules, or all of the ones that might apply to your case.  Here, too, Ulex offers an advantage, given that its rule set is relatively small and clearly demarcated.  If it is the default, you need not worry about the effects of obscure regulations issued by a government agencies you have never heard of.

Even if you figure out all the state laws and regulations that apply to your dispute, you will be hard pressed to figure out the common law rules that apply to it.  Those will depend on a vast body of case law from the relevant jurisdiction, which few lay people can locate or analyze on their own.  So they hire professionals, and wisely so.

Here, too, Ulex offers something better than conventional state law:  relying on the Restatements to sum up the common law, it had no need to cite case law at all.  Perhaps Aragon could start collecting decisions issued under its rules, giving rise to its own common law.  That's probably a good idea, in fact, and one that Luke Duncan and I have discussed.  But that would come later, and would presumably be accompanied by tools making access to Aragon case law easy.  The important point here is that Ulex could do a much better job guiding Aragon dispute resolution right now.

And make no mistake about it:  those adjudicating disputes in the Aragon network *will* want to know what rules apply.  Jurors will not achieve consensus easily you they have only their atavistic urges to rely upon in settling disputes.  Parties before them will use lawyers to craft their claims, those lawyers will search commercial databases to figure out the law, and somebody will have to pay.  Ulex is for the present at least no worse than conventional legal systems on that count and, thanks to its comparatively more compact and clear rule set, considerably better.

Once the substantive rules of Ulex escape from under the cloud of the bogus copyright claims that now hold them back, moreover, they will be available royalty free for everyone, the way the law ought to be.  Smart people are working hard on that particular problem right now.  The more quickly Ulex gets additional users, the more pressure will grow to set the law free.

**Comment by bionexus** [comment]
The way you describe Ulex it is a great way for arbitration in the classical sense. It is short and simple in comparison to the "full law" of most countries or states.

In my opinion, however, it is not the best choice for Kleros. Kleros does not aim to resolve big disputes but tries to handle claims that are too small for conventional courts. Furthermore, one key aspect of Kleros is to use jurors that are experienced with e-commerce, insurence or whatever may cause a conflict on-chain. So neither the jurors nor the parties would have access to the restatements of the common law that would be neccessary to resolve a dispute that uses Ulex as default choice of law.

I agree with you that Kleros will most likely develop its own "block chain common law" together with the policies that are passed and I believe that the source of the law used to resolve disputes should bethe principles developed in previous decisions together with the policies and the common sense of the jurors. Everything else would be hardly be any practical.

Once the restatements are freely available, I'd be glad to see them used to write some Kleros policies if neccessary.
